scripts/backport.js

```diff
@@ -18,4 +18,4 @@
  */
 
 require('../src/setup_node_env/node_version_validator');
-require('backport');
+require('../node_modules/.bin/backport');
```

Comment:
context on this?

Reply:
I just tried to standardise the way we use `node_modules` bin scripts. So in tasks there were  something like `"task1": "grunt x"` before would be now: `"task1": "kbn-node scripts/grunt x"` and a `scripts/grunt.js` was created invoking `../node_modules/.bin/grunt`. That way we could assure the `node.dev.options` would be applied on those tasks as they will run with `kbn-node`.